os_stub/cryptlib_openssl/ec_context.h

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
why ec_context.h is here, while ras_context.h, pqc_context.h is in different place?